packages/cozy-mespapiers-lib/src/components/ModelSteps/helpers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Check if a file is already selected in the state of the FormDataProvider
+ * @param {object} formData - State of the FormDataProvider
+ * @param {number} stepIndex - Used to know if the file is already selected for this step (Some paper have two Scan steps)
+ * @param {File|Blob} currentFile - File or Blob object
+ * @returns
+ */
+export const isFileAlreadySelected = (formData, stepIndex, currentFile) => {
+  for (const data of formData.data) {
+    if (data.stepIndex === stepIndex) {
+      if (currentFile.constructor.name === 'File') {
+        return (
+          data.file.name === currentFile.name &&
+          data.file.lastModified === currentFile.lastModified
+        )
```

Comment:
The next entries in `formData` will be skipped no matter if the name and the date match. Is this intended?

Reply:
Vu ensemble